port_group_test.go

```diff
@@ -17,256 +17,367 @@
 package goovn
 
 import (
-	"github.com/stretchr/testify/require"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"sort"
 	"testing"
 )
 
 const (
-	PG_TEST_PG1            = "TestPortGroup1"
-	PG_TEST_LS1           =  "TestLogicalSwitch"
-	PG_TEST_LSP1           = "TestLogicalSwitchPort1"
-	PG_TEST_LSP2           = "TestLogicalSwitchPort2"
-	PG_TEST_LSP3           = ""
-	PG_TEST_LSP1DYNADDR1   = "a.b.c.d"
-	PG_TEST_LSP2DYNADDR2   = ""
-	PG_TEST_EXT_ID_MAC_KEY = "mac_addr"
-	PG_TEST_EXT_ID_MAC     = "00:01:02:03:04:05"
-	PG_TEST_EXT_ID_MAC_2   = "01:02:03:05:05:06"
-	PG_TEST_EXT_ID_IP_KEY  = "ip_addr"
-	PG_TEST_EXT_ID_IP      = "169.254.1.1"
-	PG_TEST_OPT_1_KEY      = "foo1"
-	PG_TEST_OPT_1_VAL      = "bar1"
-	PG_TEST_OPT_1_VAL_2    = "baz1"
-	PG_TEST_OPT_2_KEY      = "foo2"
-	PG_TEST_OPT_2_VAL      = "bar2"
+	PG_TEST_PG1         = "TestPortGroup1"
```

Comment:
This commit seems to be a fix/revision of the previous one. I think it is better to be combined.

Reply:
Yup. I'll squash them.